com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: "com.deflemask.DefleMask"
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: "/app/share/DefleMask/DefleMask"
+separate-locales: false
+tags:
+  - proprietary
+
+build-options:
+    prepend-pkg-config-path: "/app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig"
```

Comment:
You probably don't need `/app/lib32` at all if you don't install anything there.

Reply:
I used the Steam config as example, fixed